### PR TITLE
Created C# class to pull daily shorts from FINRA

### PR DESCRIPTION
**Pull request by pdeneka**
Testing repo access

### CODE REVIEW

**Comment by athielen** [comment]
Might be a test, but once a language is chosen for the project, it might be good to have a repo for data ingestion pipelines from regulated sites and another for social media ingestion.

**Comment by mxgordon** [comment]
ew curly brackets on a newline

File: RegShoFINRADailyShorts.cs [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+
+
+        public void IngestSingleFileIntoMSSQL(string f)
+        {
+            try
+            {
+                //Create Connection to SQL Server
+                SqlConnection SQLConnection = new SqlConnection();
+                SQLConnection.ConnectionString = @"Data Source = DESKTOP-1786GJA\SQLEXPRESS; Database=stunks; Integrated Security=true;";
```
Comment (stevebell117) [inline RegShoFINRADailyShorts.cs:138]: This should probably be configurable.

Comment (pdeneka) [reply RegShoFINRADailyShorts.cs:138]: Was just a test for access.  Didn't secret the information that would be sensitive in a real environment.  We can actually delete this.

**Pull request closed without merging**